### ISSUE TITLE
The effectiveness of cult sacrifices are 100% dependent on *GHOST

### ISSUE THREAD

**Issue by Robustin** [tgstation/-tg-station#10316]
Many, perhaps even a majority, of players have no clue that their ability to join #TeamConstruct often depends on them remaining in their body after being murdered by a cultist. 

Some players will also Ghost just to spite you for your efforts in killing them or simply to meta-grudge. 

Not only does this completely deny you the chief purpose of a sacrifice, a soul shard, it also plays a role (in my understanding) in whether Nar-Sie grants you a word for your sacrifice. 

But this is mainly a soul shard issue. Nothing feels worse than putting in the effort to secretely abduct, murder, and gib a crewmember only to realize you got absolutely nothing for the effort. It also sucks for the many victims who are denied a chance to continue playing simply because of their ignorance of an intuitive game mechanic. 

This has been around forever, but it's literally at the top of my list of "things that are completely awful and nobody would ever object to changing them". It sucks for victims of the cult, it sucks for the cult, only person who benefits are cult-fighters and it benefits them for all the wrong reasons.

**Comment by Dorsisdwarf** [comment]
That's why you stone them the second they get into crit, Oldman. Soul stones work on anyone unconscious, not just the dead.

**Comment by SconesC** [comment]
Literally a git gud issue, soulstone them while they're in crit

If they didn't want to stick around for constructing they are probably not a good teammate either way

**Comment by phil235** [comment]
Does trying to shard someone send a message to them if they are not in their corpse? It probably should.

**Comment by Supermichael777** [comment]
to improve consistency sacrifice should not give soulstone period. construct cult is to strong to be given so easily.

**Comment by Dorsisdwarf** [comment]
I fail to see how that's consistency in any way, or related to this issue (manual soulstoning sometimes fails if you kill them all the way  because TG players are all ADHD)

**Comment by Robustin** [comment]
Yea and the only way I can get soul stones is if I can get a sacrifice without someone ghosting.

And even when I get my precious artificer, if they're braindead I'm SOL and even if they're not the cooldown on soulstones mean if I wait for a soulstone every time I want to make a new construct, the crew will literally succumb to the heat death of the universe before I can get enough soul stones to get a decent construct force rolling. 

Plus you're all focusing on the cult side of this. You're not going to see comments here from the average shitter on our server who just has no fucking clue of how cult works and deteriorates the round for both themselves and their attacker just out of ignorance. 

Given that we've added a *GET BACK IN YOUR BODY YOU SHIT* mechanic, that seems like an obvious route to take. If people would rather do that than force the ghost back into the corpse, it would have to account for the behavior of sac runes and block or have a confirmation message that prevents sac runes from triggering on soul-less bodies (since I'm pretty sure all that does anyway is give the area a nice gib shower).

**Comment by xxalpha** [comment]
Yeah this needs fixing.

**Comment by Miauw62** [comment]
oldman obviously died again but for once he's right, the shard not working if you're not in your body is pretty shitty and inconsistent.

**Comment by Robustin** [comment]
Hardly about dying. More about "I just spent 15 minutes carefully abducting the rest of my department without anyone noticing and all I have to show for it is a nice healthy layer of gibs covering every inch of my cult base. 

I know I've made that mistake myself. I get killed by a cultist and ghost out just out of curiosity to see what's going on, only to realize I've been dragged to a rune and sac'd while I was snooping around and now I've screwed myself AND the cult over.
